### ISSUE TITLE
JSON and HTML data are not playing the animation

### ISSUE THREAD

**Issue by shaikiphil** [airbnb/lottie-web#1460]
**Tell us about your environment**

* **Google Chrome Version 71.0.3578.98**
* **After Effects CC 19**

**What did you do? Please explain the steps you took before you encountered the problem.**

We were animating a simple illustration with multiple shapes that get repositioned and masked during the animation. The workflow was usual, setting up shapes from vector graphics, afterwards animating the layers. 

**What did you expect to happen?**

We have expected the export to be pretty simple, as it was in all the other AEP's for this animation set.

**What actually happened? Please include as much _relevant_ detail as possible.**

This particular animation is not playing neither as an HTML Demo, nor as a Json file. Which makes us kind of curious, because, as already mentioned, the set up and workflow were the same, as by other animations, that do work after the export. Unfortunately, I have no idea what the problem could be, since we have no trim paths and are working on a very simple motion with this document. An image folder gets exported as well anyway, even though we only got shapes and no vector graphics in this AEP. Is there something we have missed? The whole team is starting to learn how to use Bodymovin, it would be awesome if someone could help us! Thanks for everyone in advance!!!

**Please provide a download link to the After Effects file that demonstrates the problem.**

https://github.com/shaikiphil/Phil-s-AE-issue/blob/master/phil_issue.zip

or

[phil_issue.zip](https://github.com/airbnb/lottie-web/files/2759095/phil_issue.zip)

**Comment by bodymovin** [comment]
Hi, I just exported the animation and it seems to work fine.
Can you share the exported files you are getting?

**Comment by shaikiphil** [comment]
You can find our export in the attachment 

[export_phil_issue.zip](https://github.com/airbnb/lottie-web/files/2759380/export_phil_issue.zip)

**Comment by bodymovin** [comment]
Are you sure it's the same project?
I can't imagine how you could get that export from the one you shared previously.
Can you try closing and reopening AE?

**Comment by shaikiphil** [comment]
> Are you sure it's the same project?
> I can't imagine how you could get that export from the one you shared previously.
> Can you try closing and reopening AE?

Hi, we have checked the project and figured out, that the export has worked and the json was animated. However, some masks have transformed back into shapes. We have replaced them with new layers, imported from AI. So, after several attempts the animation works fine. Perhaps we have overthrown the project with some rubbish layers, that we haven't used. 

Thank you for your help!
